sway/desktop/xwayland.c

```diff
@@ -405,30 +405,19 @@ static const struct sway_view_impl view_impl = {
 	.destroy = destroy,
 };
 
-static void get_geometry(struct sway_view *view, struct wlr_box *box) {
-	box->x = box->y = 0;
-	if (view->surface) {
-		box->width = view->surface->current.width;
-		box->height = view->surface->current.height;
-	} else {
-		box->width = 0;
-		box->height = 0;
-	}
-}
-
 static void handle_commit(struct wl_listener *listener, void *data) {
 	struct sway_xwayland_view *xwayland_view =
 		wl_container_of(listener, xwayland_view, commit);
 	struct sway_view *view = &xwayland_view->view;
 	struct wlr_xwayland_surface *xsurface = view->wlr_xwayland_surface;
 	struct wlr_surface_state *state = &xsurface->surface->current;
 
-	struct wlr_box new_geo;
-	get_geometry(view, &new_geo);
+	struct wlr_box new_geo = {0};
```

Comment:
Why is it OK to not have x/y values in this box?

Reply:
Because xwayland doesn't support offset client geometry.